### DISCUSSION TITLE
Chapter 4

### DISCUSSION THREAD

**Discussion by infinite969** [Cyfrin/foundry-full-course-cu#816] [Q&A]
Hello friends, I am having trouble Accessing AggregatorV3Interface from my PriceConverter library
  Here is my code:
  // SPDX-License-Identifier: MIT
 
pragma solidity 0.8.18;
 // This imoprts Chainlink AggregatorV3 Intetrface for retriving price Data
import {PriceConverter} from "./PriceConverter.sol";
    
// ** Code instuctions **
// Recieve donation from users in ETH
// Withdraw donations
// Keep a list of donation addresses

// ETH/USD PRICE FEED ADDRESS: 0x694AA1769357215DE4FAC081bf1f309aDC325306

contract FundMe {
    using PriceConverter for uint256;
    uint256 public mininumUSD  = 5e18; // Public state variable set equal to 5.000000000000000000
    
    address[] public Funders;
    mapping (address funder => uint256 amountedFunded) public addressToAmountFunded;

    function fund() public payable  {
        require(msg.value.getConversionRate() >=mininumUSD,"didnt send enough eth"); // This requires the users donation to be greater than 5 USD
        Funders.push(msg.sender);
        addressToAmountFunded[msg.sender] = addressToAmountFunded[msg.sender] + msg.value;
    }

    
    function getPrice() public view returns(uint256) {
        AggregatorV3Interface pricefeed = AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306);
        (, int price, , ,) = pricefeed.latestRoundData();
        return uint256(price * 1e10);
    }
    

    function getConversionRate(uint256 ethAmount ) public view returns (uint256){
        uint256 ethPrice = getPrice();
        uint256 ethAmountInUSD = (ethPrice * ethAmount) / 1e18;
        return ethAmountInUSD;
    
    }   
    
    function GetVersion() public view returns (uint256)  {
        return AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306).version();
    }
}

Here is my PriceConverter Library:
/ SPDX-License-Identifier: MIT
pragma solidity 0.8.18;
import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
    

library PriceConverter{
  
    
    function getPrice() internal   view returns(uint256) {
        AggregatorV3Interface pricefeed = AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306);
        (, int price, , ,) = pricefeed.latestRoundData();
        return uint256(price * 1e10);
    }
    

    function getConversionRuate(uint256 ethAmount ) internal  view returns (uint256){
        uint256 ethPrice = getPrice();
        uint256 ethAmountInUSD = (ethPrice * ethAmount) / 1e18;
        return ethAmountInUSD;
    
    }   
}

**Comment by MoonChild362000** [comment]
Did u by any chance dint change Remix VM to Sepolia network ?

**Reply by infinite969** [reply]
both are on sepolia remix vm
